src/pages/Breeds.js

```diff
@@ -0,0 +1,66 @@
+import { Container, Row } from 'react-bootstrap';
+import { useNavigate } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import ReactLoading from 'react-loading';
+import { fetchBreeds } from '../redux/breeds';
+import { fetchCatFact } from '../redux/fact';
+import BreedCard from '../components/BreedCard';
+import cat from '../images/cat.png';
+import './breeds.css';
+
+const Breeds = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const breeds = useSelector((state) => state.breeds);
+  const fact = useSelector((state) => state.fact);
+
+  useEffect(() => {
+    if (fact === '') dispatch(fetchCatFact());
+    if (!breeds.length) dispatch(fetchBreeds());
+  }, [fetchBreeds]);
+
+  return (
+    <div id="breeds-page">
+      <div className="first-head">
+        <img src={cat} alt="cat" />
+        <p className="welcome">
+          &quot;Hi, Weclome &quot;
+        </p>
+        <div className="fact">
+          <p className="meow-fact">Cat Fact:</p>
+          {/* eslint-disable-next-line no-nested-ternary */}
+          {fact === '' ? <ReactLoading type="cubes" /> : fact ? (
+            <p>
+              &quot;
+              {fact}
+              &quot;
+            </p>
+          )
+            : <p>We failed to get a fact for meow :(</p>}
+        </div>
```

Comment:
Let's fix this text it's not well structure. You can make the image and the text side by side and the texts can be align in flex-direction column that way it will be more visible. Since this project is about mobile version kindly edit this for the mobile version.


| Your Design | Figma Design |
|-------------|---------------|
| <img width="308" alt="image" src="https://user-images.githubusercontent.com/34578631/175279710-87c7facb-7868-48fa-8ac9-3d09b016eaa4.png">| <img width="367" alt="image" src="https://user-images.githubusercontent.com/34578631/175279768-f9e56de2-0595-4fbe-b715-0b622f2ae47a.png"> |